manifests/c/CheatBreaker/CheatBreaker/2.4.2/CheatBreaker.CheatBreaker.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CheatBreaker.CheatBreaker
+PackageVersion: 2.4.2
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2023-10-19

```

Comment:
```suggestion
ReleaseDate: 2023-10-19
InstallerSwitches:
    Silent: /?
    SilentWithProgress: /?
```

Reply:
doesnt support flags, but is the same as `SilentWithProgress`